languages/java/exercises/concept/conditionals-ternary/.docs/instructions.md

```diff
@@ -0,0 +1,43 @@
+In this exercise, you'll be implementing rules for calculating the total salary of a empyoyee in a month. The International Siderurgy Company (ISC) needs help to calculate the salary for the employees, given that different factors can alter the final wage value for each employee.
+
+You have the following tasks
+
+## 1. Decide a penalty for days skipped needs to be applied
```

Comment:
Maybe rephrase this slightly as students are returning a multiplier, and not a penalty.

```suggestion
## 1. Determine the salary multiplier
```

Reply:
Makes sense, while writing i was really thinking that `penalty` was a strong term, but couldn't find a nicer one.